fdjangodog/middleware.py

```diff
@@ -1,15 +1,25 @@
 import time
 
-from datadog import statsd
+from datadog import DogStatsd
 from django.conf import settings
-from django.core.urlresolvers import resolve, Resolver404
+
+try:
+    from django.urls import resolve, Resolver404
+except ImportError:
+    from django.core.urlresolvers import resolve, Resolver404
 
 try:
     from django.utils.deprecation import MiddlewareMixin
     base_class = MiddlewareMixin
 except ImportError:
     base_class = object
 
+if hasattr(settings, 'FDJANGODOG_STATSD_HOST'):
```

Comment:
Could collapse this to a 1-liner with [getattr](https://docs.python.org/3.6/library/functions.html#getattr)